app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java

```diff
@@ -34,6 +34,8 @@
 
     private String username;
 
+    private DatabaseManager dbm;
```

Comment:
There is a Databasemanager object already created in OnCreate() We can use whichever object you think is best, but we should only have one